app/src/test/java/com/futurice/freesound/network/api/DefaultFreeSoundApiServiceTest.kt

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.network.api
+
+import com.futurice.freesound.network.api.model.AccessToken
+import com.futurice.freesound.network.api.model.SoundFields
+import com.futurice.freesound.network.api.model.SoundSearchResult
+import com.futurice.freesound.network.api.model.User
+import com.futurice.freesound.test.data.TestData
+import io.reactivex.Single
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers.*
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
+
+class DefaultFreeSoundApiServiceTest {
+
+    private val TEST_CLIENT_ID = "clientId"
```

Comment:
I was thinking the same, because the values are not accessed outside the class. I will assume for now that there's no significant performance difference.